addons/website_sale/views/templates.xml

```diff
@@ -209,8 +213,20 @@
             <div class="o_wsale_product_information position-relative d-flex flex-column flex-grow-1 flex-shrink-1">
                 <div class="o_wsale_product_information_text flex-grow-1">
                     <h6 class="o_wsale_products_item_title mb-2">
-                        <a class="text-primary text-decoration-none" itemprop="name" t-att-href="product_href" t-att-content="product.name" t-field="product.name" />
-                        <a t-if="not product.website_published" role="button" t-att-href="product_href" class="btn btn-sm btn-danger" title="This product is unpublished.">
+                        <a
+                            class="text-primary text-decoration-none"
+                            itemprop="name"
+                            t-att-href="product_href"
+                            t-att-content="product.name"
+                            t-field="product.name"
+                        />
+                        <a
+                            t-if="not product.website_published"
+                            role="button"
+                            t-att-href="product_href"
+                            class="btn btn-sm btn-danger"
+                            title="This product is unpublished."
+                        >
```

Comment:
If there is no need to touch those lines anymore, remove this part of the diff (same for the little part above)